internal/Util.hpp

```diff
@@ -214,6 +216,21 @@ static bool loadMoFileStringsToArray(FILE* moFile,
 	return true;
 }
 
+static std::vector<std::string> buildMoFilePaths(const char *domain) {
+	std::string languages = getenv("LANGUAGE");
+	std::vector<std::string> paths {""};
+	std::string slash = "/";
+	size_t pos = 0;
+	do {
+		pos = languages.find(":");
```

Comment:
Since a single character is searched I suggest to use `std::string::find(char)` instead.